lib/c-api/README.md

```diff
@@ -148,48 +148,48 @@ $ make test-capi
 
 `wasmer config` output various configuration information needed to compile programs which use Wasmer.
 
-### `wasmer config --pkg-config`
+### `wasmer config get pkg-config`
 
 It outputs the necessary details for compiling and linking a program to Wasmer,
 using the `pkg-config` format:
 
 ```bash
-$ wasmer config --pkg-config > $PKG_CONFIG_PATH/wasmer.pc
+$ wasmer config get pkg-config > $PKG_CONFIG_PATH/wasmer.pc
 ```
 
-### `wasmer config --includedir`
+### `wasmer config get includedir`
 
 Directory containing Wasmer headers:
 
 ```bash
-$ wasmer config --includedir
+$ wasmer config get includedir
```

Comment:
We shouldn't change this

Reply:
Reverted in https://github.com/wasmerio/wasmer/pull/3428/commits/710b8242f1f770271c5ab0a132f2e152047cb7dc